meteor/server/api/showStyles.ts

```diff
@@ -130,6 +151,22 @@ class ServerShowStylesAPI extends MethodContextAPI implements NewShowStylesAPI {
 	async insertShowStyleVariant(showStyleBaseId: ShowStyleBaseId) {
 		return insertShowStyleVariant(this, showStyleBaseId)
 	}
+	async insertShowStyleVariantWithBlueprint(
```

Comment:
The method is called insert**ShowStyleVariant**WithBlueprint, so why not just parse a `ShowStyleVariant` as a parameter?
(Refer to my other comment about number of parameters)

Reply:
This is implemented now.